pages/profile/[name].tsx

```diff
@@ -158,12 +158,13 @@ export const getStaticProps = async (
 ) => {
   const name = context.params?.name ?? '';
 
-  const isENS = name.endsWith('.eth');
-
   const [address, ens] = await Promise.all([
     fetchAddressFromENS(name),
     fetchENSFromAddress(name),
   ]);
+  const isENS =
+    name.endsWith('.eth') ||
```

Comment:
```suggestion
    name.endsWith('.eth') ||
```
I think we can remove this line too.

Reply:
fixed in [bf73baf](https://github.com/quest-chains/dapp/pull/183/commits/bf73bafed5b72d00ada16e497b950363fa7719cd)